code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -205,4 +221,18 @@
 	name = "pharaonic garments"
 	desc = "Ornate garments worn by the ancient rulers of Space Egypt."
 	icon_state = "pharaoh"
-	item_state = "pharaoh"
\ No newline at end of file
+	item_state = "pharaoh"
+
+/obj/item/clothing/suit/wizrobe/mystic
+	name = "mystic robes"
+	desc = "A set of thick robes made from fabric imbued with runic energies."
+	icon_state = "mysticrobe"
+	item_state = "mysticrobe"
+	heat_conductivity = SPACESUIT_HEAT_CONDUCTIVITY
```

Comment:
why is this spacesuit instead of snowgear?

Reply:
That's what winter coats have for some reason.